### PR TITLE
OCPBUGS-13346: dont log jwt tokens

### PR DESCRIPTION
**Pull request by ibihim**
# What

- Bump library-go

# Why

- Contains updated audit policies to not log tokens.

### CODE REVIEW

**Comment by stlaz** [comment]
/hold
contains _output directory

File: _output/golang-versions [hunk 1]
```diff
@@ -0,0 +1,3 @@
+1.19
```
Comment (stlaz) [inline _output/golang-versions]: please remove the `_output` dir

**Comment by ibihim** [comment]
https://github.com/openshift/cluster-authentication-operator/pull/622

**Pull request merged**